### DISCUSSION TITLE
What is your favorite food?

### DISCUSSION THREAD

**Discussion by mizuumi** [NAIST-SE/AttractingContributors#901] [Q&A]
Please help me with writing a list of favorite food!
好きな食べ物を書いて行ってください！

Link is here: 
https://github.com/mizuumi/AttractingContributors

**Comment by XelaAl** [comment]
I've always had a sweet tooth, so I vote for chocolate cake :)

**Comment by bconfessor** [comment]
I love tonkatsu, I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
